cli/src/pcluster/models/cluster.py

```diff
@@ -993,24 +990,28 @@ def get_log_events(
         :param next_token: Token for paginated requests.
         """
         if not AWSApi.instance().cfn.stack_exists(self.stack_name):
-            raise ClusterActionError(f"Cluster {self.name} does not exist")
+            raise NotFoundClusterActionError(f"Cluster {self.name} does not exist.")
 
         try:
-            if not self.stack.log_group_name:
-                raise ClusterActionError(f"CloudWatch logging is not enabled for cluster {self.name}.")
-
             log_events_response = AWSApi.instance().logs.get_log_events(
                 log_group_name=self.stack.log_group_name,
                 log_stream_name=log_stream_name,
-                end_time=isoformat_to_epoch(end_time) if end_time else None,
-                start_time=isoformat_to_epoch(start_time) if start_time else None,
+                end_time=datetime_to_epoch(end_time) if end_time else None,
+                start_time=datetime_to_epoch(start_time) if start_time else None,
                 limit=limit,
                 start_from_head=start_from_head,
                 next_token=next_token,
             )
+
             return LogStream(self.stack_name, log_stream_name, log_events_response)
         except AWSClientError as e:
-            raise _cluster_error_mapper(e, f"Unexpected error when retrieving log events: {e}")
+            if e.message.startswith("The specified log group"):
+                LOGGER.debug("Log Group %s doesn't exist.", self.stack.log_group_name)
```

Comment:
API/CLI logging level defaults to info. Would it make sense to write such log messages with info level? or we want to exclude them from the logs by default?